extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js

```diff
@@ -82,10 +82,13 @@ export const Presets = {
               const style = computedStyle(fxElement.getAmpDoc().win,
                   fxElement.getElement());
               const topAsLength = style.top == 'auto' ? '0px' : style.top;
```

Comment:
lets change this to `===` to match the others.

Reply:
Done